<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DImageViewer</class>
 <widget class="QWidget" name="DImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visilize Image</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	outline:0px;
	color:#DCDCDC;
}

*:disabled{
	background:#444444;
	border-color:#484848;
}

QWidget{
	border:1px solid #DCDCDC;
	background:#484848;
	border-radius:0px;
	color:#DCDCDC;
	border-style:none;
}

QSplitter::handle{
background:#7c7c7c;
}

QAbstractButton{
	border-radius:0px;
	color:#DCDCDC;
	background:none;
	border-style:none;
}

QAbstractButton:hover{
	color:#FFFFFF;
	background-color:#00BB9E;
}

QAbstractButton:checked,QAbstractButton:pressed{
	color:#DCDCDC;
	border-style:solid;
	border-width:0px 0px 0px 2px;
	padding:4px 4px 4px 2px;
	border-color:#00BB9E;
	background-color:#444444;
}

QLineEdit{
	border:1px solid #242424;
	border-radius:3px;
	padding:2px;
	background-color: rgb(90, 90, 90);
	color:#DCDCDC;
	selection-background-color:#484848;
	selection-background-color: rgba(81, 163, 244);
}

QLineEdit:focus{
	border:1px solid #242424;
}

QLineEdit{
	lineedit-password-character:9679;
}

.QPushButton{
	border-style:none;
	border:1px solid #242424;
	color:#DCDCDC;
	padding:5px;
	min-height:15px;
	border-radius:5px;
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

.QPushButton:hover{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
}

.QPushButton:pressed{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

QComboBox{
	border:1px solid #8f8f8f;
	background-color: rgb(118, 118, 118);
	border-radius: 5px;
}

QComboBox::down-arrow{
	image:url(:/images/qss/psblack/add_bottom.png);
	width:10px;
	height:10px;
	right:2px;
}

QComboBox::drop-down{
	subcontrol-origin:padding;
	subcontrol-position:top right;
	width:15px;
	border-left-width:0px;
	border-left-style:solid;
	border-top-right-radius:3px;
	border-bottom-right-radius:3px;
	border-left-color:#242424;
}

QComboBox::drop-down:on{
	top:1px;
}

QTabBar::tab{
border:1px solid #242424;
color:#DCDCDC;
margin:0px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
}

QTabBar::tab:selected,QTabBar::tab:hover{
border-style:solid;
border-color:#00BB9E;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

QTabBar::tab:top,QTabBar::tab:bottom{
padding:3px 8px 3px 8px;
}

QTabBar::tab:left,QTabBar::tab:right{
padding:8px 3px 8px 3px;
}

QTabBar::tab:top:selected,QTabBar::tab:top:hover{
border-width:2px 0px 0px 0px;
}

QTabBar::tab:right:selected,QTabBar::tab:right:hover{
border-width:0px 0px 0px 2px;
}

QTabBar::tab:bottom:selected,QTabBar::tab:bottom:hover{
border-width:0px 0px 2px 0px;
}

QTabBar::tab:left:selected,QTabBar::tab:left:hover{
border-width:0px 2px 0px 0px;
}

QTabBar::tab:first:top:selected,QTabBar::tab:first:top:hover,QTabBar::tab:first:bottom:selected,QTabBar::tab:first:bottom:hover{
border-left-width:1px;
border-left-color:#242424;
}

QTabBar::tab:first:left:selected,QTabBar::tab:first:left:hover,QTabBar::tab:first:right:selected,QTabBar::tab:first:right:hover{
border-top-width:1px;
border-top-color:#242424;
}

QTabBar::tab:last:top:selected,QTabBar::tab:last:top:hover,QTabBar::tab:last:bottom:selected,QTabBar::tab:last:bottom:hover{
border-right-width:1px;
border-right-color:#242424;
}

QTabBar::tab:last:left:selected,QTabBar::tab:last:left:hover,QTabBar::tab:last:right:selected,QTabBar::tab:last:right:hover{
border-bottom-width:1px;
border-bottom-color:#242424;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="mWidgetHeader" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mLabelImgColName">
        <property name="text">
         <string>Object*:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mLineEditImgName">
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string>ImgName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mLabelROI">
        <property name="text">
         <string>ROI:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mLineEditROIName">
        <property name="placeholderText">
         <string>ROI Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnSummit">
        <property name="text">
         <string>Summit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mTabLineChart">
      <attribute name="title">
       <string>Line</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="mTabBarChart">
      <attribute name="title">
       <string>Bar</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mWidgetBottom" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="mComBoxStatCata"/>
      </item>
      <item>
       <widget class="QLineEdit" name="mLineEditScale">
        <property name="placeholderText">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnDraw">
        <property name="toolTip">
         <string>Add custom index to...</string>
        </property>
        <property name="text">
         <string>Draw</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="mComBoxTheme"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
